CRUD.c

```diff
@@ -0,0 +1,186 @@
+#include <stdio.h>
+#include <stdlib.h>
```

Comment:
Check if you really need all these headerfiles or can you define functions on your own and remove unnecessary headerfiles.